MatrixSDK/Contrib/Swift/MXRestClient.swift

```diff
@@ -1027,6 +1027,32 @@ public extension MXRestClient {
         return __messages(forRoom: roomId, from: from, direction: direction.identifier, limit: _limit, filter: filter, success: currySuccess(completion), failure: curryFailure(completion))
     }
     
+    /**
+     Get list of threads for a room.
+     
+     - parameters:
+        - roomId: the id of the room.
+        - from: the token to start getting results from.
+        - direction: `MXTimelineDirectionForwards` or `MXTimelineDirectionBackwards`
+        - limit: (optional, use -1 to not defined this value) the maximum nuber of messages to return.
+        - filter: to filter returned events with.
+        - completion: A block object called when the operation completes.
+        - response: Provides a `MXPaginationResponse` object on success.
+     
+     - returns: a MXHTTPOperation instance.
+     */
+    @nonobjc @discardableResult func threads(forRoom roomId: String, from: String, direction: MXTimelineDirection, limit: UInt?, filter: MXRoomEventFilter, completion: @escaping (_ response: MXResponse<MXPaginationResponse>) -> Void) -> MXHTTPOperation {
```

Comment:
Would this be slightly clearer to read(same in MXRestClient.h/m too)? (not important though).
```suggestion
    @nonobjc @discardableResult func threads(forRoom roomId: String, from startToken: String, direction: MXTimelineDirection, limit: UInt?, filter: MXRoomEventFilter, completion: @escaping (_ response: MXResponse<MXPaginationResponse>) -> Void) -> MXHTTPOperation {
```

Reply:
Method removed.